packages/core/src/indexing/sqlite/init-tables.ts

```diff
@@ -26,3 +26,20 @@ export async function initTables(dbConnection: Knex, modelsToIndex: Array<Stream
     await createModelTable(dbConnection, tableName)
   }
 }
+
+export async function verifyTables(
```

Comment:
can it be just one function for pg and sqlite?

Reply:
Thats part of `NET-1635` to unify all possible logic.